editor/editor_file_system.cpp

```diff
@@ -1634,6 +1651,65 @@ void EditorFileSystem::_queue_update_script_class(const String &p_path) {
 	update_script_mutex.unlock();
 }
 
+void EditorFileSystem::_update_scene_groups() {
+	update_scene_mutex.lock();
+
+	for (const String &path : update_scene_paths) {
+		ProjectSettings::get_singleton()->remove_scene_groups_cache(path);
+
+		int index = -1;
+		EditorFileSystemDirectory *efd = find_file(path, &index);
+
+		if (!efd || index < 0) {
+			// The file was removed.
+			continue;
+		}
+
+		HashSet<StringName> scene_groups = _get_scene_groups(path);
```

Comment:
```suggestion
		const HashSet<StringName> scene_groups = _get_scene_groups(path);
```
The set is not modified, so it can be constant. Same for the method's return value.